### PR TITLE
[TIMOB-17493] Android: Default Translucent theme overwrites custom themes.

### PR DESCRIPTION
**Pull request by ypbnv**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-17493

**Description:**
The default Translucent theme used for activities with opacity property, modal property and incorrect background color format, overwrites any custom theme's values. 

**Test case:**
Custom theme and case provided in the ticket.

Another test case:

Add the following to tiapp.xml:
```
<android 
    xmlns:android="http://schemas.android.com/apk/res/android">
    <manifest>
      <application android:theme="@style/Theme.Titanium"/>
    </manifest>
  </android>
```
app.js
```
var win = Ti.UI.createWindow({backgroundColor: '#Z68'});
win.open();
```
As Theme.Titanium has ActionBar disabled I expected to get a translucent window without ActionBar. I got a window with an ActionBar.

**Note:**
It seems that one of the templates for generating Android manifest file is not used at all. I successfully built and SDK instance and a application without it. Removed it to avoid confusion.

### CODE REVIEW

File: android/cli/commands/_build.js [hunk 1]
```diff
@@ -3338,18 +3338,21 @@ AndroidBuilder.prototype.generateTheme = function generateTheme(next) {
 		this.logger.info(__('Generating %s', themeFile.cyan));
 
 		var flags = 'Theme.AppCompat';
+		var defaultProvidedTheme = '@style/' + flags;
```
Comment (garymathews) [inline android/cli/commands/_build.js]: You shouldn't need to change anything in this file

File: android/templates/build/theme.xml [hunk 2]
```diff
@@ -33,7 +33,13 @@
 		<!-- AppCompat Compatibility -->
 		<item name="actionBarStyle">@style/Base.Widget.AppCompat.ActionBar.Solid</item>
 	</style>
-	
+
+	<style name="Theme.AppCompat.Translucent.Default" parent="<%- defaultProvidedTheme %>">
```
Comment (garymathews) [inline android/templates/build/theme.xml]: Why not remove

```XML
<item name="android:actionBarStyle">@style/Base.Widget.AppCompat.ActionBar.Solid</item>
<item name="actionBarStyle">@style/Base.Widget.AppCompat.ActionBar.Solid</item>
```

from `Theme.AppCompat.Translucent` ?

**Review by garymathews — CHANGES_REQUESTED** [review]
CR: **SEE COMMENTS**

**Comment by ypbnv** [comment]
Closed due to being out of sync with latest build and themes changes. Will reopen once I get it back on track or I will create a new one for the issue (if it is still relevant).

**Pull request closed without merging**